src/asmcnc/apps/wifi_app/screen_wifi.py

```diff
@@ -592,20 +666,38 @@ def update_strings(self):
         self.password_label.text = self.l.get_bold("Password")
         self.country_label.text = self.l.get_bold("Country")
         self.connect_button.text = self.l.get_str("Connect")
+        self.custom_ssid_button.text = self.l.get_str("Other network")
+        self.custom_network_name.hint_text = self.l.get_str("Enter network name")
 
         self.update_font_size(self.country_label)
-        self.update_button_font_size(self.connect_button)
+        self.update_font_size(self.custom_network_name)
+        self.update_button_font_size(self.connect_button, False)
+        self.update_button_font_size(self.custom_ssid_button, True)
 
     def update_font_size(self, value):
         if len(value.text) < 8:
             value.font_size = self.default_font_size
         elif len(value.text) > 7: 
             value.font_size = self.default_font_size - 2
 
-    def update_button_font_size(self, value):
-        value.font_size = 28
-        if len(value.text) > 10:
-            value.font_size = 19
+        try:

```

Comment:
This could probably be in a separate function. Also, does the "try" negate the need for the "if value.hint_text"?